advisories/unreviewed/2022/05/GHSA-fq52-6cjf-jw59/GHSA-fq52-6cjf-jw59.json

```diff
@@ -1,23 +1,46 @@
 {
   "schema_version": "1.3.0",
   "id": "GHSA-fq52-6cjf-jw59",
-  "modified": "2022-05-24T17:22:19Z",
+  "modified": "2022-12-22T10:55:40Z",
   "published": "2022-05-24T17:22:19Z",
   "aliases": [
     "CVE-2020-2206"
   ],
-  "details": "Jenkins VncRecorder Plugin 1.25 and earlier does not escape a parameter value in the checkVncServ form validation endpoint, resulting in a reflected cross-site scripting (XSS) vulnerability.",
+  "summary": "Reflected XSS vulnerability in Jenkins VncRecorder Plugin",
+  "details": "VncRecorder Plugin 1.25 and earlier does not escape a parameter value in the `checkVncServ` form validation endpoint output.\n\nThis results in a reflected cross-site scripting (XSS) vulnerability.\n\nVncRecorder Plugin 1.35 escapes the parameter value in the output.",
   "severity": [
-
+    {
+      "type": "CVSS_V3",
+      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N"
+    }
   ],
   "affected": [
-
+    {
+      "package": {
+        "ecosystem": "Maven",
+        "name": "org.jenkins-ci.plugins:vncrecorder"
+      },
+      "ranges": [
+        {
+          "type": "ECOSYSTEM",
+          "events": [
+            {
+              "introduced": "0"
+            }
+          ]
```

Comment:
The form didn't carry over my changes for some reason. The versions affected are <= 1.25, patched since 1.35